scripts/samplesheet_generator.py

```diff
@@ -298,6 +346,18 @@ def main(lims, args):
                     os.chmod("/srv/mfs/samplesheets/{}/{}.csv".format(thisyear, obj[0]['fc']), 0664)
```

Comment:
Can you this one too ? also for all platforms. I don't think it works anyway now.